src/WishListItem/WishListItem.js

```diff
@@ -2,21 +2,42 @@ import React, { Component } from "react";
 import "./WishListItem.scss";
 
 class WishListItem extends Component {
-  constructor(props) {
-    super(props);
+  removeWishListItem = () => {
+    const [...newWishList] = this.props.wishList;
+    newWishList.splice(this.props.cardIndex, 1);
+    this.props.saveArray('wishList', newWishList);
   }
 
-  removeWishListItem = event => {
-    this.props.removeWishListItem(event.target.closest("li").id)
+  increaseWishListCount = () => {
+    if (this.props.wishListCount < 4) {
+      this.updateWishList(this.props.wishListCount + 1);
+    }
+  }
+
+  decreaseWishListCount = () => {
+    if (this.props.wishListCount > 1) {
+      this.updateWishList(this.props.wishListCount - 1);
+    }
+  }
+
+  updateWishList = (newCount) => {
+    const newWishList = this.props.wishList.map(card => {
+      if (this.props.cardName === card.cardName) {
+        card.wishListCount = newCount;
+      }
+      return card;
+    });
+    this.props.saveArray('wishList', newWishList);
   }
 
   render() {
     return(
-      <li id={this.props.cardIndex}>
+      <li id={this.props.cardIndex} className="wishlist--item">
         {this.props.cardName}
-        <i 
-          onClick={this.removeWishListItem}
-          className="far fa-trash-alt" />
+        <i className="fas fa-minus" onClick={this.decreaseWishListCount}/>
+        <span>{this.props.wishListCount}</span>
+        <i className="fas fa-plus" onClick={this.increaseWishListCount}/>
```

Comment:
I added a counter for the WishListItem, so the user can keep track of how many they want.